stuffer/s2n_stuffer_text.c

```diff
@@ -43,24 +43,27 @@ int s2n_stuffer_peek_check_for_str(struct s2n_stuffer *s2n_stuffer, const char *
     return 0;
 }
 
-int s2n_stuffer_skip_whitespace(struct s2n_stuffer *s2n_stuffer)
+int s2n_stuffer_skip_whitespace(struct s2n_stuffer *s2n_stuffer, uint32_t *skipped)
 {
-    int skipped = 0;
+    PRECONDITION_POSIX(s2n_stuffer_is_valid(s2n_stuffer));
+    uint32_t skip = 0;
     while (s2n_stuffer->read_cursor < s2n_stuffer->write_cursor) {
         switch (s2n_stuffer->blob.data[s2n_stuffer->read_cursor]) {
         case ' ':              /* We don't use isspace, because it changes under locales */
         case '\t':
         case '\n':
         case '\r':
             s2n_stuffer->read_cursor += 1;
-            skipped += 1;
+            skip += 1;
             break;
         default:
-            return skipped;
+            goto finished;
```

Comment:
why use a `goto` when `break` would work just as well?

Reply:
in this case, it was to avoid duplicated code https://github.com/awslabs/s2n/pull/2037#discussion_r448981031

Reply:
`break` would serve the same purpose, and avoid using `goto`

Reply:
i don't think so, unfortunately... since `break` means something in in the context of `switch`